controllers/topolvm_controller.go

```diff
@@ -129,20 +129,25 @@ func getControllerDeployment(lvmCluster *lvmv1alpha1.LVMCluster, namespace strin
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      TopolvmControllerDeploymentName,
 			Namespace: namespace,
+			Labels: map[string]string{
+				DefaultLabelKey: TopolvmControllerLabelVal,
+			},
 		},
 		Spec: appsv1.DeploymentSpec{
 			Replicas: &replicas,
 			Selector: &metav1.LabelSelector{
 				MatchLabels: map[string]string{
-					AppAttr: TopolvmControllerDeploymentName,
+					AppAttr:         TopolvmControllerDeploymentName,
```

Comment:
Are we using this existing label anywhere? If not I would like to remove it.

Reply:
can be removed